packages/js-sdk/tests/sandbox/host.test.ts

```diff
@@ -21,3 +24,35 @@ sandboxTest('ping server in sandbox', async ({ sandbox }) => {
     }
   }
 })
+
+sandboxTest('ping server in non-running sandbox', async () => {
+  const sbx = await Sandbox.create(template, { timeoutMs: 60_000 })
```

Comment:
Use the sandbox from the test case
```suggestion
sandboxTest('ping server in non-running sandbox', async ({ sandbox }) => {
```

Reply:
it's ok if i kill it?

Reply:
Is there a reason why to create it manually, the one from test case should be handled even though the test fails anywhere during execution, you don't want to care about such thing in the test